provider/import.go

```diff
@@ -163,7 +163,9 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 	}
 
 	if hcl != nil {
-		hcl.Interpolate(interpolation)
+		if hcl.GetOptions().Interpolate {
+			hcl.Interpolate(interpolation)
+		}
```

Comment:
Ok I see that you only do it for HCL but the docs say it'll also do it for State IIRC?

Also do not do it like this, this should be on the implementation, not the caller, that's why are options of the Writer, not the Importer.